docs/sources/tempo/api_docs/_index.md

```diff
@@ -422,6 +406,23 @@ $ curl http://localhost:3200/api/v2/search/tag/.service.name/values | jq .
 }
 ```
 
+#### Filtered tag values
+If you set Tempo's `autocomplete_filtering_enabled` configuration parameter to `true` (default value is `false`), you can provide an optional URL query parameter, `q` to your request. `q` is a Url encoded [TraceQL query]({{< relref "../traceql" >}}. If provided, the tag values returned by the API are filtered to only return values seen on spans matching your filter parameters. 
```

Comment:
Having a lowercase q start a sentence looks weird, so I've added "The parameter". I've made each sentence it's own line for ease of updates.
```suggestion
If you set Tempo's `autocomplete_filtering_enabled` configuration parameter to `true` (default value is `false`), you can provide an optional URL query parameter, `q` to your request.
The `q` parameter is a Url encoded [TraceQL query]({{< relref "../traceql" >}}.
If provided, the tag values returned by the API are filtered to only return values seen on spans matching your filter parameters. 
```